devenv/datasources.yaml

```diff
@@ -253,6 +255,39 @@ datasources:
           target:
             expr: "{ job=\"test\" }"
           field: "traceID"
+      - targetUID: WyFv5154z
+        label: "Provision test 1"
+        description: "this is a test correlation from provisioning"
+        config:
+          type: query
+          target:
+            editorMode: "code"
+            format: "table"
+            rawQuery: "true"
+            rawSql: "SELECT * FROM superhero WHERE name='$${path}' AND eyecolor='$${__data.fields.traceID}'"
```

Comment:
This is to demonstrate the use case of showing fieldPaths and will be removed before final approval.